### ISSUE TITLE
filler - broken game engine binary 

### ISSUE THREAD

**Issue by Yoows** [01-edu/public#2590]
smart-road
---

The game_engine doesn't work on MacOS (Intel based)
I get this error : ./linux_game_engine: cannot execute binary file or ./m1_game_engine: cannot execute binary file

**Comment by bomanviktor** [comment]
@nprimo  He is referring to the second project [filler](https://github.com/01-edu/public/tree/master/subjects/filler).

**Comment by nprimo** [comment]
@bomanviktor are you encountering the same issue? Could you (and @Yoows) please share more information about the hardware/OS that cannot run the binaries provided?
We'll try to provide an alternative as soon as possible - I'm currently not experiencing any issue with the following:

```console
$ uname -a
Linux pop-os 6.8.0-76060800daily20240311-generic #202403110203~1715181801~22.04~aba43ee SMP PREEMPT_DYNAMIC Wed M x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by bomanviktor** [comment]
@nprimo No, I have already passed the project last year. We had an issue at grit:lab where we had no game engine for ARM64 architecture but it was resolved in #2231.

**Comment by nprimo** [comment]
@Yoows please, feel free to comment here with more information if the issue persists.
